transifex_catalogs/fa.json

```diff
@@ -1,50 +1,102 @@
+  "exchanges_category_fiat": "پول نقد (Fiat)",
+  "exchanges_category_high_liquidity": "با نقدینگی بالا",
+  "exchanges_category_instant_exchange": "صرافی های آنی",
+  "exchanges_description": "مبادلات ارز دیجیتال روش آسانی برای خرید و معامله ی دیکْرِد(DCR) فراهم می‌آورد. سلب مسئولیت: قبل از استفاده از هر سرویسی، همیشه خودتان درباره آن تحقیق کنید (DYOR). ایمن‌ترین روش برای ذخیره کردن دیکْرِد(Decred) در کیف پولی است که در آن منحصراً خود شما کلیدهای خصوصی را کنترل می‌کنید.",
+  "exchanges_disclaimer": "به جز دی سی آر دکس <a target='_blank' rel='noopener noreferrer' href='https://dex.decred.org/'>DCRDEX</a> هیچکدام از صرافی های لیست شده بالا به کاربران اجازه مالکیت دارایی هایشان را در طی مراحل معاملات نمیدهند.بعضی از صرافی های متمرکز، شامل نوع آنی (instant) نیز ممکن است حتی بعد از واریز دارایی ها، برای تکمیل معاملات، اطلاعات شخصی درخواست کنند. به کاربران اکیداً توصیه میشود جهت افزایش امنیت و حریم خصوصی دارایی هایشان بر بستر دی-سی-آر دکس ترید کنند. <a target='_blank' rel='noopener noreferrer' href='https://www.youtube.com/watch?v=XaE_xzGKjeQ'>راهنمای دی-سی-آر دکس</a> موجود است.",
+  "exchanges_payment_title": "پردازشگران پرداخت دیکْرِد(Decred)",
+  "exchanges_title": "صرافی ها",
+  "general_copyright": "توسعه دهندگان دیکْرِد",
+  "general_javascript_warning": "جاوااسکریپت لازم است.",
+  "general_javascript_warning_description": "متأسفیم، اما این وب‌سایت بدون فعال بودن جاوااسکریپت درست کار نمی‌کند.",
+  "home_download_wallet": "دانلود کیف پول",
+  "landing_freedom_headline": "آزادی",
+  "landing_freedom_link1": "قانون اساسی دیکْرِد(Decred) را بخوانید",
+  "landing_freedom_text": "هیچکسی نمیتواند جلوی انتقال پولت را بگیرد.",
+  "landing_insights_headline": "نگرش های قدرتمند",
+  "landing_insights_link1": "بازدید از Dcrdata Block Explorer",
+  "landing_insights_text": "تصور کن چطور اجزا در پیوند با هم قرار میگیرند.",
+  "landing_opener": "پول تکامل یافت",
+  "landing_privacy_headline": "حریم خصوصی",
+  "landing_privacy_link1": "یاد بگیر و دیکْرِدت را میکس کن",
+  "landing_privacy_link2": "مشاهده مشارکت در حریم خصوصی دیکْرِد(Decred)",
+  "landing_privacy_text": "هیچکس به جز خودت نباید بداند که با پولت چکار میکنی.",
+  "landing_resolve_headline": "رفع مناقشه",
+  "landing_resolve_link1": "رای دادن از نوع (POS)",
+  "landing_resolve_text": "روش حل اختلاف تعبیه شده تا مردم در هماهنگی با هم زندگی کنند.",
+  "landing_seemore": "بیشتر",
+  "landing_sovereignty_headline": "حاکمیت",
+  "landing_sovereignty_link1": "اداره حکومت دیکْرِد(Decred)",
+  "landing_sovereignty_link2": "تغییرات در اجماع را دنبال کنید",
+  "landing_sovereignty_text": "شما قوانین را تعیین می کنید و اجرا می کنید.",
+  "landing_treasury_headline": "خزانه (Treasury)",
+  "landing_treasury_link1": "مشارکت و تبدیل شدن به کارگزار",
+  "landing_treasury_text": "توانایی توسعه پایدار.",
+  "nav_become_contributor": "یک مشارکت کننده شوید",
+  "nav_brand_resources": "منابع نام تجاری",
+  "nav_bugbounty": "Bug Bounty Program",
+  "nav_community": "انجمن",
+  "nav_dcrdata": "Block Explorer",

```

Comment:
Missed string?

Reply:
The same as before. changed to Farsi orthography.